mllib/src/test/scala/org/apache/spark/ml/PipelineSuite.scala

```diff
@@ -230,7 +230,9 @@ class PipelineSuite extends SparkFunSuite with MLlibTestSparkContext with Defaul
 }
 
 
-/** Used to test [[Pipeline]] with [[MLWritable]] stages */
+/**
+ * Used to test [[Pipeline]] with `MLWritable` stages
```

Comment:
We should avoid inlined comment when there are code blocks (`` ` ... ` ``). See https://github.com/apache/spark/pull/16050